timings.c

```diff
@@ -18,6 +18,26 @@
 #define	MAX(a,b) (((a)>(b))?(a):(b))
 #endif	/* MAX */
 
+unsigned long probe_timing_cache(char *adrs) {
```

Comment:
It took me a a bit to notice that this function excludes the `clflush`. Let's add a comment here making note of that.